docs/.vuepress/public/presets/toml/bracketed-segments.toml

```
@@ -164,7 +164,7 @@ format = '\[[$symbol($version)]($style)\]'
 format = '\[[$symbol$environment]($style)\]'
 
 [sudo]
-format = '\[[as $symbol]\]'
+format = '\[[as $symbol($style)]\]'
```

With your change, the syntax was still invalid:
```suggestion
format = '\[[as $symbol]($style)\]'
```